scripts/postinstall.ts

```diff
@@ -117,7 +117,8 @@ const GULPDELTA_FILEPATH = path.join(CURR_DIR, '..', 'resources', 'gulpfile.delt
 const GULPDELTA_CONTENT = fs.readFileSync(GULPDELTA_FILEPATH, 'utf8');
 
 // update file contents
-fs.writeFileSync(GULPFILE_FILEPATH, gulpFileData.replace(/build.initialize\(require\('gulp'\)\);/g, GULPDELTA_CONTENT));
+if (!(gulpFileData.indexOf(GULPDELTA_CONTENT) >= 0)) fs.writeFileSync(GULPFILE_FILEPATH, gulpFileData.replace(/build.initialize\(require\('gulp'\)\);/g, GULPDELTA_CONTENT));
```

Comment:
Why is this needed? Confused... you'd only ever install this NPM package on a project once, so that task wouldn't be present. So why add the extra complexity to check if it is present?

Asking in an effort to understand why this is being suggested... I can't see/replicate a scenario where this would happen IRL.